class-wc-facebookcommerce.php

```diff
@@ -368,6 +368,21 @@ public function add_wordpress_integration() {
 			new WP_Facebook_Integration();
 		}
 
+		/**
+		 * Saves errors or messages to WooCommerce Log (woocommerce/logs/plugin-id-xxx.txt)
+		 *
+		 * @since 2.3.1-dev.1
```

Comment:
We'll need to update the @`since` tag to `2.3.3` before merging.

Reply:
Hi @rcstr !
Tks for validating the fix.
I've just updated the since first as requested.